compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -369,9 +370,9 @@ getPerDimLayoutAttrs(MLIRContext *context, TileSwizzle swizzle) {
           PerDimLayoutAttr::get(context, labels[1], shape[1])};
 };
 
-static ConcreteMmaLayout getConcreteMFMALayout(MLIRContext *context,
-                                               MMAIntrinsic intrinsic) {
-  auto opaque = getOpaqueMFMALayout(context, intrinsic);
+static ConcreteMmaLayout getConcreteMMALayout(MLIRContext *context,
```

Comment:
Quickly merge this PR, because @Groverkss said he would make a PR deleting `getConcreteMMALayout` tomorrow ;-)